nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -2001,6 +2026,44 @@
         propertyData.setItems([]);
     };
 
+    /**
+     * Gets all the referenced parameters from the {parameterContext} based on the value of {parameterReference}
+     *
+     * @param {string} parameterReference
+     * @returns {ParameterEntity[]}
+     */
+    var getExistingParametersReferenced = function (parameterReference) {
```

Comment:
It also looks like the Property Table already has a `containsParameterReference` function. Can it be used instead? I see that one is simply detecting while the other is extracting the Parameter Reference. If there's no opportunity for reuse that's ok. Just wanted to raise awareness here in case.